.github/workflows/acceptance.yml

```diff
@@ -14,7 +22,39 @@ on:
   pull_request:
 
 jobs:
+  changes:
```

Comment:
All workflows below will depend on this one.
We will not trigger real workflows or Visual Snapshot if there are no FE or BE changes.